sparrow_datums/boxes/frame_augmented_boxes.py

```diff
@@ -86,7 +86,11 @@ def to_darwin_file(
         path: str = "/",
         label_names: Optional[list[str]] = None,
     ) -> None:
-        """Write Darwin annotation dict to disk."""
+        """
+        'output_path' is path and filename for saving the Darwin-formatted file.
+        'filename' is image/video name that matches Darwin dataset.
+        'label_names' can be list or tuple of strings.
+        """
```

Comment:
Just a couple style things here:

* I like to use the NumPy docstring style. That should include the summary sentence at the top and then a slightly different format for the [parameters](https://numpydoc.readthedocs.io/en/latest/format.html#parameters).
* You'll need a blank line between the summary and the rest of the description. You're currently failing one of the style checks in the [automatic build](https://github.com/sparrowml/sparrow-datums/runs/7023283676?check_suite_focus=true). Once you make the change, you can validate that the style is correct by running `make codestyle` to make changes automatically or `make check-codestyle` to see if everything is kosher.